### DISCUSSION TITLE
I can't connect to the internet after I connected to the vpn server

### DISCUSSION THREAD

**Discussion by gabrielbalinca** [angristan/openvpn-install#887] [Help]
Hi !

As the title say's the vpn client can't no longer connect to the internet after it connects to the vpn server.

The setup :

Server VM :

OS : Debian 10 Buster
Docker Version : 20.10.6
Container : ubuntu 20.04
I build the contaner with this command :

docker run -it -d -p 1194:1194/udp --name sovpn --cap-add=NET_ADMIN ubuntu /bin/bash

In the container I run these commands :

apt update -y
apt install -y curl iproute2

cd /dev
mkdir net
mknod net/tun c 10 200
chmod 0666 net/tun

curl -O https://raw.githubusercontent.com/angristan/openvpn-install/master/openvpn-install.sh
chmod +x openvpn-install.sh
./openvpn-install.sh

Client VM :
OS: Windows 10

Both VMs are on the same machine. Any idea why client vm lose's it's internet connection after the connection it's established ?

EDIT : I found the issue, this thread can be removed.

**Comment by enboig** [comment]
Could you post the solution? I have a similar problem using Ubuntu

**Reply by gabrielbalinca** [reply]
Hi! I'm sorry but I can't remember what was the issue, I abandoned that project for now.
